src/pages/project.jsx

```diff
@@ -0,0 +1,47 @@
+import {useParams} from "react-router-dom";
+import DefaultTemplate from "../components/templates/DefaultTemplate";
+import Button from "../components/atoms/Button";
+import Table from "../components/atoms/Table";
+import Th from "../components/atoms/Th";
+import Td from "../components/atoms/Td";
+import { useProject } from "../lib/hooks/states/project";
+
+const ProjectPage = () => {
+  const { id } = useParams();
+  const { tasks } = useProject({ id });
+
+  return (
+    <DefaultTemplate>
+      <h1>Tasks</h1>
+      <Table>
+        <thead>
+          <tr>
+            <Th>Title</Th>
+            <Th>Description</Th>
+            <Th>Created At</Th>
+          </tr>
+        </thead>
+        <tbody>
+          {tasks.map(({ title, description, createdAt }) => (
+            <tr>
```

Comment:
You should add key props for <tr>
```suggestion
            <tr key={taskId}>
```